test/elpa.el

```diff
@@ -0,0 +1,5 @@
+(setq package-user-dir
+      (expand-file-name (format ".cask/%s/elpa" emacs-version)))
```

Comment:
> I like the idea of Cask. The problem with it is it often fails to do its job (current error is there's no package s- when downloading dependencies).

I'm reasonably sure this was some bug in `package.el` on Emacs's master branch. I thought this was resolved in newer builds.